ddtrace/internal/flare.py

```diff
@@ -30,22 +30,23 @@
 
 
 class Flare:
-    def __init__(self, timeout_sec: int = DEFAULT_TIMEOUT_SECONDS):
-        self.original_log_level = 0  # NOTSET
-        self.timeout = timeout_sec
+    def __init__(self, timeout_sec: int = DEFAULT_TIMEOUT_SECONDS, flare_dir: str = TRACER_FLARE_DIRECTORY):
+        self.original_log_level: int = logging.NOTSET
+        self.timeout: int = timeout_sec
+        self.flare_dir: pathlib.Path = pathlib.Path(flare_dir)
```

Comment:
Is this something that can be configured by the user? If not, could we perhaps use a [temporary directory](https://docs.python.org/3/library/tempfile.html#tempfile.TemporaryDirectory) instead?

Reply:
I could maybe use a temp dir just for the tests, but not sure if I can do it for actual tracer flares. I thought about using the temp dir, but I was worried that in the case a customer needs to trigger a tracer flare, they would probably be restarting their app one or more times, in which case the generated files would disappear.